net45 fails on macOS

I am trying to build a simple project:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFrameworks>net45</TargetFrameworks>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.2.0" PrivateAssets="All" />
  </ItemGroup>
  <Import Project="$(MSBuildSDKExtrasTargets)" Condition="Exists('$(MSBuildSDKExtrasTargets)')" />
</Project>
```

There is an error:

> error MSB4019: The imported project "/Library/Frameworks/Mono.framework/Versions/5.4.1/lib/mono/xbuild/15.0/Bin/Microsoft.CSharp.targets" was not found. 

To fix this, I added this directly after the `<Import>`:

```xml
<PropertyGroup Condition=" '$(TargetFramework)' == 'net45' and '$(OS)' != 'Windows_NT' ">
  <LanguageTargets>$(MSBuildToolsPath)\Microsoft.CSharp.targets</LanguageTargets>
</PropertyGroup>
```

--- a/MSBuild.Sdk.Extras/build/platforms/NETFramework.targets
+++ b/MSBuild.Sdk.Extras/build/platforms/NETFramework.targets
@@ -3,7 +3,7 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
-  <Import Project="$(MSBuildThisFileDirectory)languageTargets\Wpf.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)languageTargets\Wpf.targets" Condition=" '$(OS)' == 'Windows_NT' " />
 
   <!-- Settings only apply to .NET Framework -->
   <ItemGroup Condition="'$(EnableDefaultItems)' == 'True' and '$(EnableDefaultSettingsItems)' == 'true' ">

